mars/scheduler/tests/test_assigner.py

```diff
@@ -41,7 +41,7 @@ def get_reply(self):
 class Test(unittest.TestCase):
 
     def testAssignerActor(self):
-        with create_actor_pool(backend='gevent') as pool:
+        with create_actor_pool(n_process=1, backend='gevent') as pool:
```

Comment:
@wjsi Do we have any process ability to test on the AssignerActor? Is it fine to create only a process?

Reply:
AssignerActor does not rely on processes

Reply:
good, resolved